cypress/e2e/features/Ui/linkTest.feature

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
you can move this scenario to the Double Vanities feature file. Let s try to keep all scenarios in same feature file for one page